addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -10,89 +10,87 @@
-                        <div t-if="showDelete and !ui.isSmall"
-                            class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
-                            tabindex="0" aria-label="Remove" role="menuitem" t-on-click.stop="() => this.onClickUnlink(attachment)" title="Remove">
-                            <i class="fa fa-trash"/>
+            }" role="menu" >
+                <t t-foreach="this.props.attachments" t-as="attachment" t-key="attachment.id">
+                    <div t-if="attachment.isImage"
+                        t-att-aria-label="attachment.filename"
+                        class="o-mail-AttachmentImage d-flex position-relative flex-shrink-0 mw-100 mb-1 me-1"
+                        t-att-title="attachment.name"
+                        t-att-class="{ 'o-isUploading': attachment.uploading }"
+                        tabindex="0"
+                        t-att-data-mimetype="attachment.mimetype"
+                        t-on-click="() => this.fileViewer.open(attachment, props.attachments)"
+                        role="menuitem"
+                    >
+                        <img
+                            class="img img-fluid my-0 mx-auto o-viewable"
+                            t-att-class="{ 'opacity-25': attachment.uploading }"
+                            t-att-src="getImageUrl(attachment)"
+                            t-att-alt="attachment.name"
+                            t-attf-style="max-width: min(100%, {{ imagesWidth }}px); max-height: {{ props.imagesHeight }}px;"
+                            t-on-load="onImageLoaded"
+                        />
+                        <div t-if="attachment.uploading" class="position-absolute top-0 bottom-0 start-0 end-0 d-flex align-items-center justify-content-center" title="Uploading">
+                            <i class="fa fa-spin fa-spinner"/>
                         </div>
-                        <div t-if="canDownload(attachment) and !ui.isSmall" class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover mt-auto" t-on-click.stop="() => this.onClickDownload(attachment)" title="Download">
-                            <i class="fa fa-download"/>
+                        <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
+                            <div t-if="showDelete"
+                                class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
+                                tabindex="0" aria-label="Remove" role="menuitem" t-on-click.stop="() => this.onClickUnlink(attachment)" title="Remove">
+                                <i class="fa fa-trash"/>
+                            </div>
+                            <div t-if="canDownload(attachment)" class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover mt-auto" t-on-click.stop="() => this.onClickDownload(attachment)" title="Download">
```

Comment:
same remark here, the condition on `and !ui.isSmall` is removed

Reply:
added it back.